src/pages/clients-list/forms/ClientTemplateForm.js

```diff
@@ -605,6 +587,37 @@ const ClientTemplateForm = ({ setErrorMessage, recordId, _labels, plantId, maxAc
       setProfessionFilterStore(filteredList)
     }
   }
+  const { userId } = JSON.parse(window.sessionStorage.getItem('userData'))
+
+  const onClose = async () => {
+    const values = clientIndividualFormik.values
+    try {
+      const { record: cashAccountRecord } = await getRequest({
+        extension: `SY.asmx/getUD`,
+        parameters: `_userId=${userId}&_key=cashAccountId`
+      })
+
+      const data = {
+        recordId: values?.recordId || null,
+        functionId: 3600,
+        plantId: plantId || values?.plantId,
+        reference: values?.reference,
+        clientId: values.clientId,
+        status: values.status,
+        cashAccountId: cashAccountRecord.value
```

Comment:
what is that ? 
cashAccount should be taken from the form, like plant
if we don't have it then we don't need it